src/Api/OAuthApi.php

```diff
@@ -0,0 +1,267 @@
+     * @param array $clientId Array ['SANDBOX' => '', 'PRODUCTION' => ''] of application keys App ID (Client ID).
+     * @param array $clientSecret Array ['SANDBOX' => '', 'PRODUCTION' => ''] of application keys Cert ID (Client Secret).
+     * @return OAuthToken|null
+     * @throws ApiException on non-2xx response
+     */
+    public function getClientToken(
+        array $clientId,
+        array $clientSecret
+    ): ?OAuthToken {
+        $response = $this->getOAuthWithHttpInfo(
+            $clientId,
+            $clientSecret,
+            GrantTypeEnum::CLIENT_CREDENTIALS
+        );
+        return $response['data'] ?? null;
+    }
+
+    /**
+     * Operation getAuthorizationToken
+     *
+     * <p>When a user grants your application the authorization to take action on their behalf, eBay returns an <i>authorization code </i> that contains the user's consent for the specified scopes. Use the authorization code in a <code>POST</code> request that's commonly known as an <i>authorization code grant request</i>. This request gets a User access token and its associated refresh token. </p><p>For more info see <a href="https://developer.ebay.com/api-docs/static/oauth-auth-code-grant-request.html">Exchanging the authorization code for a User access token</a></p><p>When you mint a new User access token, the access token is returned along with a <i>refresh token</i>, which you can use to renew the User access token for the associated user. A <b>refresh token request</b> mints an access token that contains the same authorization properties as the original access token.</p><h2 id="about">About refresh tokens</h2><p>For security, a User access token is short-lived. However, a refresh token is long-lived and you can use it to renew a User access token after the token expires. The benefit is that you don't need to get the account-owner's consent each time you need to renew their User access token.</p>
+     * @param array $clientId Array ['SANDBOX' => '', 'PRODUCTION' => ''] of application keys App ID (Client ID).
+     * @param array $clientSecret Array ['SANDBOX' => '', 'PRODUCTION' => ''] of application keys Cert ID (Client Secret).
+     * @param string $redirectUri The 'RuName' value assigned to your application and the environment you're targeting.
+     * @param string $code The URL-encoded authorization code value returned to you by eBay when the user granted their consent.
+     * @return OAuthToken|null
+     * @throws ApiException on non-2xx response
+     */
+    public function getAuthorizationToken(
+        array $clientId,
+        array $clientSecret,
+        string $redirectUri,
+        string $code,
+    ): ?OAuthToken {
+        $tokenFile = __DIR__ . DIRECTORY_SEPARATOR . ".." . DIRECTORY_SEPARATOR . ".." . DIRECTORY_SEPARATOR ."tokenData.info";
+
+        if(file_exists($tokenFile)) {
```

Comment:
@Edogusma we use PSR-12 coding style throughout our projects, please apply it in this PR: https://www.php-fig.org/psr/psr-12/
```suggestion
        if (file_exists($tokenFile)) {
```

Reply:
Got it. Updated.

Reply:
@Edogusma there are still a few places that are not updated